src/main/scala/edu/berkeley/ce/sparkrocks/SeedJointSelector.scala

```diff
@@ -160,12 +165,14 @@ object SeedJointSelector {
       } else {
         // Joint meets criteria
         val remainingVolume = jointOption.get
-        if (remainingVolume.volume <= volumePerPiece) {
+        val nonRedundantVolume = Block(remainingVolume.center, remainingVolume.nonRedundantFaces)
```

Comment:
Okay, let me ask another question to see if I'm understanding this change correctly.
Before, we used the volume of the remaining rock mass to determine what to do, but now we also need the volume of the new seed block we've just created.
That's why we didn't have this code to remove non-redundant faces before, but now we need it, right?